README.md

```diff
@@ -310,6 +310,7 @@ available from the Electron API.  See the following options for usage.
     
     -w | --outputWait          Integer – Time to wait (in MS) between page load and PDF creation.  
                                          If used in conjunction with -e this will override the default timeout of 10 seconds    
+    -ignoreCertificateErrors  Boolean - Whether to accept self-signed and untrusted certificates
```

Comment:
Can you please fix this so it has two dashes? (`--ignoreCertificateErrors`)
I also think it would be more clear if it said:
```
--ignoreCertificateErrors  Boolean - If true, all certificate errors thrown by Electron will be ignored.  This can be used to accept self-signed and untrusted certificates.  You should be aware of the security implications of setting this flag.
                             false - default
```

`options.js` also needs to be updated yet.

![image](https://user-images.githubusercontent.com/12678528/66831542-e9643e00-ef25-11e9-88ca-da556f5452db.png)